app/konnect/dev-portal/register-and-create-app.md

```diff
@@ -0,0 +1,79 @@
+---
+title: Register and create an application as a developer
+---
+
+This guide explains how developers can get started with the developer platform by registering and creating an application. 
+
+Dev Portal allows developers to create applications by using APIs from another company. For example, say you're a developer for a company that wants to consume two different APIs from another company. To make this possible, you'd need to create an application in Dev Portal that consumes both the APIs.
+
+The following diagram shows how you can register your app in Dev Portal:
+
+{% mermaid %}
+flowchart TD
+    A[Sign up for Dev Portal] -->|Access approved| B(Create an application)
+    B --> |Registration approved| C(Add products to the application)
+    C --> D(Create application credentials)
+    D --> |Generate in Dev Portal| E[Use application] 
+    D --> |Create in IdP| E
+{% endmermaid %}
```

Comment:
```suggestion
{% mermaid %}
flowchart TD
    A[Sign up for Dev Portal] --> B(Create an application)
    B --> C(Add products to the application)
    C --> D(Create application credentials)
    D --> |Generate in Dev Portal| E[Use application] 
    D --> |Create in IdP| E
{% endmermaid %}
```

Because the approval steps are outside of the control of the dev reading the doc, I think they can be removed.